frontend/rievent-frontend/src/components/Register.js

```diff
@@ -1,94 +1,153 @@
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+          },
         }
-    };
+      );
 
-    return (
-    <><Navbar /><div className="Register-div">
+      //history.push('/');
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return (
+    <>
+      <Navbar />
+      <div className={classes.form}>
+        <Typography variant="h4" gutterBottom align='center' fontFamily='Roboto '>
+          Register
+        </Typography>
         <form onSubmit={handleSubmit}>
-        <label htmlFor="fullName">Full Name</label>
-        <input
-            type="text"
+          <CustomTextField
+            className={classes.textField}
+            label="Full Name"
+            variant="outlined"  
             id="fullName"
             name="fullName"
-            placeholder="Enter your full name"
             value={fullName}
-            onChange={(event) => setFullName(event.target.value)} />
-
-        <label htmlFor="email">Email</label>
-        <input
+            onChange={(event) => setFullName(event.target.value)} 
+            marginBottom = {classes.textField}          
+          />
+          <CustomTextField
+            className={classes.textField}
+            label="Email"
+            variant="outlined"
             type="email"
             id="email"
             name="email"
-            placeholder="Enter your email address"
             value={email}
-            onChange={(event) => setEmail(event.target.value)} />
-
-        <label htmlFor="password">Password</label>
-        <input
+            onChange={(event) => setEmail(event.target.value)}
+            sx={{ marginBottom: '0px' }}
+          />
+          <CustomTextField
+            className={classes.textField}
+            label="Password"
+            variant="outlined"
             type="password"
             id="password"
             name="password"
-            placeholder="Enter your password"
             value={password}
-            onChange={(event) => setPassword(event.target.value)} />
-
-        <label htmlFor="confirmPassword">Confirm Password</label>
-        <input
+            onChange={(event) => setPassword(event.target.value)}
+          />
+          <CustomTextField
+            className={classes.textField}
+            label="Confirm Password"
+            variant="outlined"
             type="password"
             id="confirmPassword"
             name="confirmPassword"
-            placeholder="Confirm your password"
             value={confirmPassword}
-            onChange={(event) => setConfirmPassword(event.target.value)} />
-
-        <button type="submit">Register</button>
-        <p>
-            <Link to="/login">Already have an account? Log in!</Link>
-        </p>
-        </form>
-    </div></>
-    );
-};
-
-export default Register;
+            onChange={(event) => setConfirmPassword(event.target.value)}
+          />
+          <Button
```

Comment:
Button component's tags have to aligned. Buttons are inside the form, so closing form tag needs less indentation than Button tag. In general, apply indentation  so that element's hierarchy is clear.